ape_safe/accounts.py

```diff
@@ -102,8 +108,9 @@ def fallback_handler(self) -> Optional[ContractInstance]:
 
     @cached_property
     def client(self) -> SafeClient:
-        if self.provider.chain_id not in self.account_file["deployed_chain_ids"]:
-            raise  # Not valid on this chain
```

Comment:
@dtdang This would just cause `No exception on the stack to raise` to pop up in Python which is a bit weird, so that is why we are changing these to be more meaningful

Reply:
Here is what I mean:

<img width="609" alt="Screenshot 2023-09-07 at 12 32 49" src="https://github.com/ApeWorX/ape-safe/assets/19540978/6fa01062-f65a-4404-a36c-725c4f32fce4">

Reply:
Got it. Thank you.